src/components/video/Video.vue

```diff
@@ -309,9 +309,6 @@ export default {
                         if (args.action === 'modal-opened') {
                             this.playVideo();
                         }
-                        if (args.action === 'modal-closed') {
-                            this.reRenderVideo();
-                        }
```

Comment:
I'm slightly confused by this (not the change, the original code). As far as I know this was always meant to fire on modal open rather than close, to force the auto play to work properly, and I can't tell why the change would have been made (by me). There might have been a reason that's just slipping my mind now but who knows. Just to check, with this removed, does it still autoplay properly when you open a modal with a video in it? Ideally repeatedly, when you open then close then reopen

Reply:
Ignoring the issue where the audio just keeps playing in the background sometimes when you close it which is always fun, it looks like it does - although now it continues playing from where you closed it rather than from the beginning. That's probably fine, we can just keep an eye on whether some oddness about that doesn't come back, and good if this fixes the actual issue here

Reply:
Yeah, @chesterroberts-vs and I discussed the same thing, we weren't sure why it would be re-rendering the video on close! I checked live and when I open the modal and close it, the video & sound appears to stop only when I hit the close button. If I click the background outside of the modal, it continues.  

I think as part of this change we should add the functionality to stop the video when the modal is hidden rather than on click of the close button.

Reply:
I have updated it so that the video should now pause when the modal is closed by either clicking the close button or pressing outside the modal